src/python/tests/core/platforms/android/stack_analyzer_test.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for the stack analyzer module for android specifically."""
+
+import os
+import unittest
+
+from base import utils
+from crash_analysis.stack_parsing import stack_analyzer
+from system import environment
+from tests.test_libs import helpers as test_helpers
+
+DATA_DIRECTORY = os.path.join(os.path.dirname(__file__), 'stack_analyzer_data')
+TEST_JOB_NAME = 'test'
+
+KERNEL_REPRO = """kernel/build u'c059b39e1caf2b96aa376582eeb93062b43d69d5'
```

Comment:
I need to understand how you plan to hook this information for srcmap/link hyperlinking. Will you change stacktrace lines to include this in next CL (i remember you said part 2).

Reply:
Hmm, I need to look at how the srcmap code works, I will probably extract the correct hash and project from this.  Yeah the next cl will use this data